### ISSUE TITLE
Etebase-server fails with PydanticUserError

### ISSUE THREAD

**Issue by plustik** [NixOS/nixpkgs#291257]
### Describe the bug
After updating NixOS etebase-server fails to start and instead outputs an error of type pydantic.errors.PydanticUserError:
```console
[user@system:~]# journalctl -u etebase-server.service
Jan 19 15:27:05 main-nixos-helsinki systemd[1]: Starting An Etebase (EteSync 2.0) server...
Jan 19 15:27:08 main-nixos-helsinki etebase-server-pre-start[1704899]: Operations to perform:
Jan 19 15:27:08 main-nixos-helsinki etebase-server-pre-start[1704899]:   Apply all migrations: admin, auth, contenttypes, django_etebase, myauth, sessions, token_auth
Jan 19 15:27:08 main-nixos-helsinki etebase-server-pre-start[1704899]: Running migrations:
Jan 19 15:27:08 main-nixos-helsinki etebase-server-pre-start[1704899]:   No migrations to apply.
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/responsive_rtl.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/responsive.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/rtl.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/login.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/changelists.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/fonts.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/forms.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/nav_sidebar.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/autocomplete.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/dashboard.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/widgets.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/base.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/vendor/select2/select2.min.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/vendor/select2/LICENSE-SELECT2.md'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/css/vendor/select2/select2.css'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-unknown-alt.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-no.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/selector-icons.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-calendar.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/LICENSE'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-unknown.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/README.txt'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/tooltag-arrowright.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/search.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-addlink.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-changelink.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/tooltag-add.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-alert.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-yes.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-deletelink.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/sorting-icons.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-viewlink.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/calendar-icons.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/icon-clock.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/inline-delete.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/gis/move_vertex_off.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/img/gis/move_vertex_on.svg'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/fonts/README.txt'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/fonts/LICENSE.txt'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/fonts/Roboto-Bold-webfont.woff'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/fonts/Roboto-Regular-webfont.woff'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/fonts/Roboto-Light-webfont.woff'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/popup_response.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/urlify.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/jquery.init.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/SelectBox.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/actions.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/collapse.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/nav_sidebar.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/prepopulate.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/cancel.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/prepopulate_init.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/core.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/SelectFilter2.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/calendar.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/inlines.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/change_form.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/autocomplete.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/jquery/jquery.min.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/jquery/LICENSE.txt'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/jquery/jquery.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/LICENSE.md'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/select2.full.min.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/select2.full.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/hu.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/sr-Cyrl.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ja.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/fi.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ca.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/nl.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/th.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/bs.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/uk.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/zh-TW.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ru.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/sl.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/sk.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/pt.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/lv.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/hi.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ms.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/af.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/km.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/tr.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ar.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/tk.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/az.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/sq.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/bn.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/nb.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/dsb.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/is.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/pt-BR.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/lt.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/sv.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/vi.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/he.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/it.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/mk.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/fr.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/da.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/id.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ka.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/en.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/el.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/hy.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/fa.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/bg.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ps.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/zh-CN.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/hsb.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/es.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/de.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/pl.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/gl.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/hr.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/cs.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ne.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/sr.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ko.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/eu.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/ro.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/select2/i18n/et.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/xregexp/xregexp.min.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/xregexp/xregexp.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/vendor/xregexp/LICENSE.txt'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/admin/RelatedObjectLookups.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: Deleting 'admin/js/admin/DateTimeShortcuts.js'
Jan 19 15:27:09 main-nixos-helsinki etebase-server-pre-start[1705830]: 128 static files copied to '/var/etebase/static'.
Jan 19 15:27:09 main-nixos-helsinki systemd[1]: Started An Etebase (EteSync 2.0) server.
Jan 19 15:27:13 main-nixos-helsinki etebase-server-start[1705955]: /nix/store/c7y7a9p6cv9fsp39fqm84wglqlc1kip0-python3-3.11.7-env/lib/python3.11/site-packages/pydantic/_internal/_config.py:321: UserWarning: Valid config keys have changed in V2:
Jan 19 15:27:13 main-nixos-helsinki etebase-server-start[1705955]: * 'orm_mode' has been renamed to 'from_attributes'
Jan 19 15:27:13 main-nixos-helsinki etebase-server-start[1705955]:   warnings.warn(message, UserWarning)
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]: /nix/store/c7y7a9p6cv9fsp39fqm84wglqlc1kip0-python3-3.11.7-env/lib/python3.11/site-packages/pydantic/_internal/_config.py:321: UserWarning: Valid config keys have changed in V2:
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]: * 'keep_untouched' has been renamed to 'ignored_types'
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   warnings.warn(message, UserWarning)
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]: Traceback (most recent call last):
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/cl01cal3vnrxm2pjzagmijv5sri8fdrv-python3.11-daphne-4.0.0/bin/.daphne-wrapped", line 9, in <module>
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     sys.exit(CommandLineInterface.entrypoint())
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/c7y7a9p6cv9fsp39fqm84wglqlc1kip0-python3-3.11.7-env/lib/python3.11/site-packages/daphne/cli.py", line 171, in entrypoint
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     cls().run(sys.argv[1:])
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/c7y7a9p6cv9fsp39fqm84wglqlc1kip0-python3-3.11.7-env/lib/python3.11/site-packages/daphne/cli.py", line 233, in run
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     application = import_by_path(args.application)
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/c7y7a9p6cv9fsp39fqm84wglqlc1kip0-python3-3.11.7-env/lib/python3.11/site-packages/daphne/utils.py", line 12, in import_by_path
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     target = importlib.import_module(module_path)
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/asiphbpiy2gmidfm3xbwcikayhs66289-python3-3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     return _bootstrap._gcd_import(name[level:], package, level)
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/ajc9hkc5yqpqb3ijc2vgmysbwicaqmqr-python3.11-etebase-server-0.11.0/lib/etebase-server/etebase_server/asgi.py", line 19, in <module>
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     application = create_application()
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:                   ^^^^^^^^^^^^^^^^^^^^
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/ajc9hkc5yqpqb3ijc2vgmysbwicaqmqr-python3.11-etebase-server-0.11.0/lib/etebase-server/etebase_server/asgi.py", line 10, in create_application
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     from etebase_server.fastapi.main import create_application
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/ajc9hkc5yqpqb3ijc2vgmysbwicaqmqr-python3.11-etebase-server-0.11.0/lib/etebase-server/etebase_server/fastapi/main.py", line 13, in <module>
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     from .routers.authentication import authentication_router
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/ajc9hkc5yqpqb3ijc2vgmysbwicaqmqr-python3.11-etebase-server-0.11.0/lib/etebase-server/etebase_server/fastapi/routers/authentication.py", line 84, in <module>
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     class Login(Authentication):
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/c7y7a9p6cv9fsp39fqm84wglqlc1kip0-python3-3.11.7-env/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 92, in __new__
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     private_attributes = inspect_namespace(
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:                          ^^^^^^^^^^^^^^^^^^
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:   File "/nix/store/c7y7a9p6cv9fsp39fqm84wglqlc1kip0-python3-3.11.7-env/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 372, in inspect_namespace
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]:     raise PydanticUserError(
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]: pydantic.errors.PydanticUserError: A non-annotated attribute was detected: `response_data = <django.utils.functional.cached_property object at 0x7fe3b2cadc10>`. All model fields require a type annotation; if `response_data` is not meant to be a field, you may be able to resolve this error by annotating it as a `ClassVar` or updating `model_config['ignored_types']`.
Jan 19 15:27:14 main-nixos-helsinki etebase-server-start[1705955]: For further information visit https://errors.pydantic.dev/2.5/u/model-field-missing-annotation
Jan 19 15:27:14 main-nixos-helsinki systemd[1]: etebase-server.service: Main process exited, code=exited, status=1/FAILURE
Jan 19 15:27:14 main-nixos-helsinki systemd[1]: etebase-server.service: Failed with result 'exit-code'.
```

### Steps To Reproduce
1. Set `services.etebase-server = true;` in the NixOS-config.
2. rebuild and switch to new system (restart etebase-server.service)
3. Show status of etebase-server.service (systemctl shows status `failed`)

### Expected behavior
Systemctl should show status `active (running)` for etebase-server.service.

### Additional context
The first time the error appears in my logs was on 19th January.
This is probably the problem discussed in https://github.com/etesync/server/issues/170. I think etebase-server is currently incompatible with the version of fastapi in nixpkgs. As long as it's not predictable when etebase-server will update it's dependency on fastapi, I propose to replace fastapi v0.109.0 with fastapi v0.103.1 in the inputs of etebase-server. I think v0.103.1 is the last compatible version of fastapi, but I'm not sure.

### Notify maintainers

@felschr 
@wd15 (because of fastapi)

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.73, NixOS, 24.05 (Uakari), 24.05.20240118.7f10f17`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by plustik** [comment]
I tried to fix the problem in this branch: https://github.com/plustik/nixpkgs/tree/fix/EtebaseServerWrongFastapi, but the tests of fastapi keep failing, probably because other python-modules, that fastapi depends on, were updated since fastapi v0.103.1 was in nixpkgs and are now also incompatible with fastapi v0.103.1.

**Comment by felschr** [comment]
Should be fixed by #283319

**Comment by plustik** [comment]
I can confirm that etebase-server works again (at least for me). Thanks to everyone involved.
